### PR TITLE
Update Issue/PR templates

### PR DESCRIPTION
**Pull request by Cam0Cow**
PR and issue ticket templates

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md
[the remark this answers is not in the document]
Comment (IshDeshpa) [reply PULL_REQUEST_TEMPLATE.md:1]: Test has been tested on hardware

**Review by IshDeshpa — CHANGES_REQUESTED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 
+
+## Requirements 
+- [ ] Followed Coding Style Guide 
+- [ ] Code Build checks pass 
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it
+- [ ] Software feature has documentation for it updated
+- [ ] Test provides useful information and uses relevant data to accurately represent Controls 
+- [ ] Tested on hardware
```
Comment (IshDeshpa) [inline PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
- [ ] Tests have passed on hardware
```

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 
+
+## Requirements 
+- [ ] Followed Coding Style Guide 
+- [ ] Code Build checks pass 
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it
+- [ ] Software feature has documentation for it updated
```
Comment (IshDeshpa) [inline PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
- [ ] Software feature has documentation for it updated in both ReadTheDocs and the code
```

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Quality Assurance Checklist 
+To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 
+
+## Requirements 
+- [ ] Followed Coding Style Guide
```
Comment (IshDeshpa) [inline PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
- [ ] Followed Coding Style Guide 
- [ ] Presented/discussed in some capacity with others on the Controls team
```

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] Followed Coding Style Guide 
+- [ ] Code Build checks pass 
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it
+- [ ] Software feature has documentation for it updated
+- [ ] Test provides useful information and uses relevant data to accurately represent Controls 
+- [ ] Tested on hardware
+- NOTE: If test file already exists, use that one
+- [ ] If applicable, have you added the new feature to main.c
+- [ ] Did you have fun?
```
Comment (IshDeshpa) [inline PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
- [ ] Tagged appropriate issue ticket on this PR
- [ ] Did you have fun?
```

File: PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- [ ] Followed Coding Style Guide 
+- [ ] Presented/discussed in some capacity with others on the Controls team
+- [ ] Code Build checks pass 
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it
+- [ ] Software feature has documentation for it updated in both ReadTheDocs and the comments
+- [ ] Test provides useful information and uses relevant data to accurately represent Controls 
+- [ ] Tests have passed on hardware
+- NOTE: If test file already exists, use that one
+- [ ] If applicable, have you added the new feature to main.c
```
Comment (Arnav-Surjan) [inline PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
- [ ] If applicable, have you added the new feature to main.c?
```
Add question mark.

**Review by Arnav-Surjan — CHANGES_REQUESTED** [review]
Everything looks good aside from the changes I suggested.

File: PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+## Requirements 
+- [ ] Followed Coding Style Guide 
+- [ ] Presented/discussed in some capacity with others on the Controls team
+- [ ] Code Build checks pass 
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it
+- [ ] Software feature has documentation for it updated in both ReadTheDocs and the comments
+- [ ] Test provides useful information and uses relevant data to accurately represent Controls 
+- [ ] Tests have passed on hardware
+- NOTE: If test file already exists, use that one
```
[the remark this answers is not in the document]
Comment (IshDeshpa) [reply PULL_REQUEST_TEMPLATE.md:15]: "Testing" perhaps?

**Review by IshDeshpa — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Which feature is being tested?
+2. Which requirements are being tested?
+3. Which test is used?
+4. What is the hardware configuration?
+5. What combination of inputs cause the error?
+6. What does the error look like?
```
Comment (IshDeshpa) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]:
```suggestion
6. What does the error look like?
7. Does this issue already exist?
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest a new feature for Controls
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Describe the problem this feature will solve**
+A clear and concise description of what the problem is.
```
Comment (IshDeshpa) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]:
```suggestion
A clear and concise description of what the problem is. Please make sure the issue for this feature does not already exist.
```

**Review by NathanielDelgado — APPROVED** [review]
lgtm

**Review by NathanielDelgado — APPROVED** [review]
Really good work team!

**Review by IshDeshpa — APPROVED** [review]
LGTM 💅💅💅💅

**Review by NathanielDelgado — APPROVED** [review]
Love to see it!♥

**Pull request merged**